src/schemas/microservice.js

```diff
@@ -0,0 +1,75 @@
+const microserviceCreate = {
+  "id": "/microserviceCreate",
+  "type": "object",
+  "properties": {
+    "name": {"type": "string"},
+    "config": {"type": "string"},
+    "catalogItemId": {"type": "integer"},
+    "isNetwork" : {"type": "boolean"},
+    "needUpdate" : {"type": "boolean"},
+    "rebuild": {"type": "boolean"},
+    "flowId": {"type": "integer"},
+    "ioFogNodeId": {"type": "string"},
+    "rootHostAccess": {"type": "boolean"},
+    "deleteWithCleanUp": {"type": "boolean"},
+    "logSize": {"type": "integer"},
+    "imageSnapshot": {"type": "string"},
+    "volumeMappings": {
+      "type": "array",
+      "items": {"$ref": "volumeMappings"}},
+    "ports": {"$ref": "/ports"},
+    "routes": {
+       "type": "array",
+       "items": {"type": "string"}}
+    },
+  "required": ["name"]
+};
+
+const microserviceUpdate = {
+  "id": "/microserviceUpdate",
+  "type": "object",
+  "properties": {
+    "name": {"type": "string"},
+    "config": {"type": "string"},
+    "isNetwork" : {"type": "boolean"},
+    "needUpdate" : {"type": "boolean"},
+    "rebuild": {"type": "boolean"},
+    "ioFogNodeId": {"type": "string"},
+    "rootHostAccess": {"type": "boolean"},
+    "deleteWithCleanUp": {"type": "boolean"},
```

Comment:
maybe it's better to move it to microserviceDelete

Reply:
do i need to validate data in delete? there is no data except id as usual